pkg/i2gw/providers/kong/kong.go

```diff
@@ -51,25 +54,23 @@ func NewProvider(conf *i2gw.ProviderConf) i2gw.Provider {
 // ToGatewayAPI converts the received i2gw.InputResources to i2gw.GatewayResources
 // including the kong specific features.
 func (p *Provider) ToGatewayAPI(_ i2gw.InputResources) (i2gw.GatewayResources, field.ErrorList) {
-	return p.converter.convert(p.storage)
+	return p.convert(p.storage)
```

Comment:
why moving a way from the common approach across the current implementations? 

We have a converter and the the converter is the one to have the method `convert`

Reply:
no particular reason, there are equivalent. Reverted to be coherent with other implementations.